lib/node_modules/@stdlib/math/generics/random/sample/test/test.factory.js

```diff
@@ -110,7 +110,44 @@ tape( 'the function allows setting a default `replace` strategy for the returned
 	t.equal( actual.length, 3, 'returned array has three elements' );
 	t.deepEqual( actual, expected, 'returned array is a random draw' );
 
-	expected = [ 1, 3, 5 ];
+	expected = [ 4, 2, 5 ];
+	actual = sample({
+		'size': 3
+	});
+	t.equal( actual.length, 3, 'returned array has three elements' );
+	t.deepEqual( actual, expected, 'returned array is a random draw' );
+
+	t.end();
+});
+
+tape( 'the function allows setting a default `mutate` strategy for the returned function', function test( t ) {
+	var expected;
+	var sample;
+	var actual;
+
+	sample = factory( [ 1, 2, 3, 4, 5, 6 ], {
+		'replace': false,
+		'mutate': true,
+		'seed': 223
+	});
+
+	expected = [ 2, 5, 6 ];
+	actual = sample({
+		'size': 3
+	});
+	t.equal( actual.length, 3, 'returned array has three elements' );
+	t.deepEqual( actual, expected, 'returned array is a random draw' );
+
+	// Strategy can be overriden...
+	expected = [ 4, 1, 3 ];
```

Comment:
How does this test currently demonstrate that a strategy can be overridden? Currently, the test combines both mutation and overriding which is not obvious from the test description.

Instead, I would create a new (separate) test 

> the default `mutate` strategy can be overridden

with the same basic structure as this. And to make more clear that the default strategy can be overridden, set the default `mutate` to `false`. Then for each invocation, set `mutate` to `true` until `null` is returned.